server/router/api/v1/memo_service.go

```diff
@@ -363,6 +368,17 @@ func (s *APIV1Service) DeleteMemo(ctx context.Context, request *v1pb.DeleteMemoR
 		}
 	}
 
+	// Delete memo comments
+	relations, err := s.Store.ListMemoRelations(ctx, &store.FindMemoRelation{RelatedMemoID: &id})
```

Comment:
`Type: MemoRelationComment`

```suggestion
	relations, err := s.Store.ListMemoRelations(ctx, &store.FindMemoRelation{RelatedMemoID: &id})
```

Reply:
Not sure what you are asking here? The suggested change looks identical to the existing code?

Reply:
Your comment removes the memo comment, but there is no relation type specified here?

Reply:
Ah I see what you mean. Sorted so it targets the comments - also added in logic to delete the any references to this deleted Memo